installer/src/main/bin/encryptModel.cmd

```diff
@@ -21,298 +21,64 @@
 @rem This script uses the following command-line arguments directly, the rest
 @rem of the arguments are passed down to the underlying python program:
 @rem
-@rem     - -oracle_home        The directory of the existing Oracle Home to use.
-@rem                           This directory must exist and it is the caller^'s
-@rem                           responsibility to verify that it does. This
-@rem                           argument is required.
-@rem
-@rem     - -domain_type        The type of domain to create. This argument is
-@rem                           is optional.  If not specified, it defaults to WLS.
-@rem
-@rem     - -wlst_path          The path to the Oracle Home product directory under
-@rem                           which to find the wlst.cmd script.  This is only
-@rem                           needed for pre-12.2.1 upper stack products like SOA.
-@rem
-@rem                           For example, for SOA 12.1.3, -wlst_path should be
-@rem                           specified as %ORACLE_HOME%\soa
+@rem     -oracle_home
```

Comment:
CER has suggested getting rid of the "The script uses... directly" section in these files.  I agree, it doesn't add any value, and adds confusion about what the full set of arguments includes.  Opinions?

Reply:
If we put anything up here it should only be the environment variables specific to the script. These aren't described in the usage statement.